### ISSUE TITLE
[Feature Request]: Support animated cover 

### ISSUE THREAD

**Issue by NischalDawadi** [mierak/rmpc#86]
I have few animated album covers from apple music.
It will be awesome if you can support it I tried using gif but it didn't work.

**Comment by mierak** [comment]
So I took a quick look at things.
* rmpc with ueberzugp already supports animations
* inline image protocol does support gifs and we should be able to implement it relatively easily
* kitty, from the look of things, wants us to send every frame of the animation independently and then it can manage the animation itself, so not bad at all, should be fairly straightforward to implement as well
* and last, sixel, seems like we would have to control rendering of each separate frame ourselves, which while possible, does not seem to match all that well with the immediate mode of ratatui so im leaning to say no here

tldr: ueberzugpp works, kitty and inline image protocol will support animations when i get to it and will most likely leave animations unsupported on sixel

**Comment by mierak** [comment]
Animated GIFs are now supported with https://github.com/mierak/rmpc/pull/94 merged.
Do note though, that they work only with Kitty and Iterm2 image protocol, with Iterm2 protocol having additional limitations in regards to gif size in Tmux which we cannot do anything about.
https://mierak.github.io/rmpc/configuration/album-art/#iterm2-inline-image-protocol.
